sklearn/neighbors/tests/test_dist_metrics.py

```diff
@@ -121,6 +121,29 @@ def haversine_slow(x1, x2):
                               np.sin(0.5 * D2) ** 2)
 
 
+def test_jensenshannon_metric():
```

Comment:
With scikit-learn conventions, this should be named 'test_jensen_shannon_metric'